### ISSUE TITLE
Add new todo item on pressing enter in between todo list rows

### ISSUE THREAD

**Issue by kamalkishor1991** [upnotes-io/react-todo#24]
Pressing enter for creating new row works for the first item but it does not work for rest of the rows. 
https://upnotes-io.github.io/react-todo/

1. If cursor is in the middle the line should get split into 2 with dividing the text into 2 lines and cursor remains in the next line.
2. By pressing enter in the end of the line should cause new row to be created in the next line and the curser shoud be in the next line.
3. By pressing enter in the begining of the line should cause the row to be created at the top and the cursor should remain in the current line.

Basically 2ed and 3ed points are special conditions to the first point so if first usecase is implemented correctally the 2ed and 3ed should automatically work.

**Comment by yeswanthannadata** [comment]
Hi @kamalkishor1991, I want to work on this. Please assign it to me.

**Comment by serickson67** [comment]
Hi @kamalkishor1991 - I also wouldn't mind giving this issue a try. Please assign it to me if you are okay with me working on it. Thanks!

**Comment by yeswanthannadata** [comment]
#26 @kamalkishor1991 @bhujoshi

**Comment by bhujoshi** [comment]
@yeswanthannadata Thanks. Added comments on PR.

**Comment by yeswanthannadata** [comment]
@bhujoshi Thanks. I will check and update

**Comment by raj5036** [comment]
Hi @kamalkishor1991 , can I work on this?

**Comment by kamalkishor1991** [comment]
@raj5036 sure. assigning to you.

**Comment by kamalkishor1991** [comment]
can you check `git remote -v` in your repository? I think your origin is set to `https://github.com/upnotes-io/react-todo.git` instead of the the fork. 
https://stackoverflow.com/questions/42830557/git-remote-add-origin-vs-remote-set-url-origin
You should change the origin to your fork.

**Comment by gkasoff** [comment]
> can you check `git remote -v` in your repository? I think your origin is set to `https://github.com/upnotes-io/react-todo.git` instead of the the fork. https://stackoverflow.com/questions/42830557/git-remote-add-origin-vs-remote-set-url-origin You should change the origin to your fork.

Thank you for helping with that. PR sent.

**Comment by gkasoff** [comment]
Hi @kamalkishor1991 can you review my PR request?

**Comment by Jahsid** [comment]
@kamalkishor1991  Could you please assign it to me?

**Comment by rampravesh19-96** [comment]
Hi, I would like to work on this issue. Could you please assign it to me?
